CMakeLists.txt

```diff
@@ -276,7 +285,7 @@ endif()
 #
 find_program(
   CLANG_FORMAT_EXECUTABLE
-  NAMES clang-format clang-format-3.9 clang-format-3.8
+  NAMES clang-format-3.8
```

Comment:
- Why did you remove support for other versions of `clang-format`?
- Could we use `clang-format-${LLVM_VERSION}` to make this slightly more portable?
- Even better, does Clang install a `Config.cmake` file that includes the path to this executable?